version/1/4/nl/code_of_conduct.md

```diff
@@ -0,0 +1,73 @@
+# Gedragscode voor Bijdragers
+
+## Onze belofte
+
+In het belang van het bevorderen van een open en welkomende omgeving, beloven wij,
+de bijdragers en onderhouders, om het deelnemen in ons project en onze gemeenschap
+een intimidatie-vrije ervaring te maken voor iedereen. Dit onafhankelijk van leeftijd,
+lichaam, handicap, etniciteit, gender identiteit en uiting, ervaringsniveau, nationaliteit,
```

Comment:
Hey Anouk,

First of all thank you for reviewing. With regards to 'gender' vs 'geslacht' I purposefully choose 'gender', mainly because of this: http://www.gelijkekansenbrussel.irisnet.be/nl/gender/2eme-genre/sexe-genre/
Would you agree with this?